src/audio/pcm_converter/pcm_converter_generic.c

```diff
@@ -176,7 +185,16 @@ static int32_t _pcm_convert_f_to_i(int32_t src)
 	return dst;
 }
 
-static int32_t _pcm_convert_i_to_f(int32_t src)
+/**
+ * \brief convert fixed number to float
+ *
+ * \param src integer number to convert
+ * \param pow additional exponent component
+ *            number of fractional bits in fixed point value.
+ *            Use '0' for normal conversion to float
+ * \return (int32_t)(src * 2**pow)
+ */
```

Comment:
Documentation for this function feels odd, given the implementation. The function body does the useful thing, but the description... feels inverted.